app/components/views/CreateGame.vue

```diff
@@ -38,26 +38,51 @@
     import Vue from "nativescript-vue";
     import RadDataForm from "nativescript-ui-dataform/vue";
     import PickerField from 'nativescript-picker/vue';
+    import { SocketIO } from 'nativescript-socketio';
     import axios from 'axios'
- 
+    
     Vue.use(PickerField);
     Vue.use(RadDataForm);
 
     export default {
         methods: {
+            handleCreateClick(){
+            var testSocket = new SocketIO('https://2fa9f776.ngrok.io');
+            let userTypedInCode = 'game1'
+            // get game data
+            let gameInfo = {
+                lat: "90",
+                long: "90",
+                markerLimit: 3,
+                playerLimit: 2,
+                timeLimit: 100,
+                startTime: 20,
+                code: userTypedInCode,
+            }
+            // make request to server save a game to the DB (sending game info)
+            axios.post('https://2fa9f776.ngrok.io/createGame', gameInfo)
+            .then((result) => {
+                testSocket.connect();
```

Comment:
This is great, just remember to change this from testSocket to something like 'socket'